File: Code Python/Exercicio-8/sequencia-numero-inteiro-while.py
#aula-25-09-19

num = int(input("Digite o numero inteiro diferente de 0: "))
soma = int(0)
while num != 0:
    soma = soma + num
    num = int(input("Digite outro numero: "))
print(f"Resultado da soma dos numero inteiro eh: {soma}")

File: Code_C/Exercicio-07/sequencia-numero-inteiro-while.c
#include <stdio.h>
#include <stdlib.h>

void pausar(){
	printf("\nPressione alguma tecla para continuar...");
	getch();
}

int main(){
	int numint, soma = 0;

	printf("Digite um numero inteiro:  ");
	scanf("%d", &numint);

	while (numint != 0){
		soma = soma + numint;
		scanf("%d", &numint);
	}
	printf("Resultado da soma dos numeros inteiros eh: %d \n", soma);

	pausar();
	return 0;
}
